specs/consul/SOURCES/consul-server.sysconfig

```diff
@@ -0,0 +1,2 @@
+# Path to configuration file
+CONF_FILE="/etc/consul/server/config.json"
```

Comment:
Don't forget about the new lines at the end of the files.

Reply:
Fixed (0a34c6cfcaf9d82bb710a77d8e28e4c36016bfa3).